mkdocs/tests/utils_tests.py

```diff
@@ -67,3 +68,34 @@ def test_create_media_urls(self):
         for path, expected_result in expected_results.items():
             urls = utils.create_media_urls(site_navigation, [path])
             self.assertEqual(urls[0], expected_result)
+
```

Comment:
Yeah, I was just about to correct about the Py31 to Py27.  But yeah, it would have been much cleaner.